Card/src/androidTest/java/com/braintreepayments/api/card/CardClientTest.java

```diff
@@ -199,7 +208,7 @@ public void tokenize_callsErrorCallbackForInvalidCvv() throws Exception {
         sut.tokenize(card, (cardResult) -> {
             assertTrue(cardResult instanceof CardResult.Failure);
             Exception error = ((CardResult.Failure) cardResult).getError();
-            assertEquals("CVV verification failed",
+            Assert.assertEquals("CVV verification failed",
```

Comment:
```suggestion
           assertEquals("CVV verification failed",
```
🧶 : This happens sometimes when the imports get rearranged. The `Assert` part shouldn't be needed.

Reply:
good catch!